Lib/test/test_heapq.py

```diff
@@ -135,6 +153,13 @@ def test_heappushpop(self):
         x = self.module.heappushpop(h, 11)
         self.assertEqual((h, x), ([11], 10))
 
+    def test_heappop_max(self):
```

Comment:
Actually, this already uses self.module, so I think this is does apply to/pass for both the C and Python code.